spaces/S000143/README.md

```diff
@@ -0,0 +1,17 @@
+---
+uid: S000143
+name: Butterfly space
+aliases:
+- Bow-tie space
+refs:
+  - zb: "0148.16701"
+    name: $\aleph_0$-spaces (E. Michael)
+  - zb: "0684.54001"
+    name: General Topology (Engelking, 1989)
+---
+
+The space $X$ is the closed upper half-plane topologized as follows.  Points above the $x$-axis have their usual Euclidean neighborhoods.  A local base for a point $p$ on the $x$-axis is formed by the "butterfly neighborhoods" $N_\varepsilon(p)$ for $\varepsilon>0$, where $N_\varepsilon(p)$ consists of $p$ together with all points $q$ at distance less than $\varepsilon$ from $p$ and lying underneath the union of the two rays in $X$ that emanate from $p$ and have slopes $\varepsilon$ and $-\varepsilon$.
```

Comment:
```suggestion
The space $X=\mathbb R\times [0,\infty)$ is the closed upper half-plane topologized as follows. Points above the $x$-axis have their usual Euclidean neighborhoods. A local base for a point $p=(p_x,0)$ on the $x$-axis is formed by the "butterfly neighborhoods" $N_\varepsilon(p)$ for $\varepsilon>0$, where $$N_\varepsilon(p)=X\cap B_\varepsilon(p)\setminus\{(x,y):y\geq \varepsilon|x-p_x|\}$$ consists of $p$ together with all points $q$ at distance less than $\varepsilon$ from $p$ and lying underneath the union of the two rays in $X$ that emanate from $p$ and have slopes $\varepsilon$ and $-\varepsilon$.
```

![image](https://github.com/user-attachments/assets/2652ee92-579d-49e4-bf4e-3a586331888b)

(assuming I understood the definition)

Reply:
The suggested formula does not quite work, as it omits the point $p$.  But I'll fix this.